pkg/cloudprovider/providers/vsphere/vsphere_test.go

```diff
@@ -515,18 +515,25 @@ func TestInstances(t *testing.T) {
 	t.Logf("Found InstanceID(%s) = %s\n", nodeName, instanceID)
 
 	_, err = i.InstanceID(context.TODO(), nonExistingVM)
-	if err == cloudprovider.InstanceNotFound {
```

Comment:
I don't think this should change. Why is the typed error not returned any more?

Reply:
thanks, let me take another look

Reply:
@liggitt you are right, i end up fix an issue at InstanceID() implementation to make the test pass,
with that, i don't need to change `cloudprovider.InstanceNotFound`